packages/babel-plugin-transform-es2015-parameters/README.md

```diff
@@ -2,6 +2,12 @@
 
 > Compile ES2015 default and rest parameters to ES5
 
+This plugin transforms ES2015 parameters to ES5, this includes:
```

Comment:
Are we going to remove this from the website readme then? It already shows this currently